gradle/libs.versions.toml

```diff
@@ -1,21 +1,20 @@
 [versions]
-agp = "8.1.4"
-kotlin = "1.9.22"
+agp = "8.3.2"
```

Comment:
why is not the latest?

Reply:
I thought it was compatible with AS, but looks like it's ok. Thank you